test/web/namer_web_test.py

```diff
@@ -41,6 +39,10 @@ def chrome_factory(debug: bool) -> WebDriver:
 
 def edge_factory(debug: bool) -> WebDriver:
     options = EdgeOptions()
+    if (system() == 'Linux' and os.environ.get("DISPLAY") is None) or not debug:
+        options.headless = True
+    if system() != 'Windows' and os.geteuid() == 0:
+        options.add_argument("--no-sandbox")
```

Comment:
Do this actually work in your env?   I can't test it locally.
Browser should run headless unless you are debugging.

Reply:
It's worked before and yes, it's still working